Makefile

```diff
@@ -213,13 +213,14 @@ haskell_dir            := $(DEFN_DIR)/haskell
 haskell_files          := $(ALL_FILES)
 haskell_main_module    := ETHEREUM-SIMULATION
 haskell_syntax_module  := $(haskell_main_module)
-haskell_main_file      := driver
-haskell_kompiled       := $(haskell_dir)/$(haskell_main_file)-kompiled/definition.kore
+haskell_main_file      := driver.md
+haskell_main_filename  := $(basename $(notdir $(haskell_main_file)))
+haskell_kompiled       := $(haskell_dir)/$(haskell_main_filename)-kompiled/definition.kore
 
 build-haskell: $(haskell_kompiled)
 
 $(haskell_kompiled): $(haskell_files)
-	$(KOMPILE_HASKELL) $(haskell_main_file).md                  \
+	$(KOMPILE_HASKELL) $(haskell_main_file)                     \
```

Comment:
Yes, let's be consistent across all the definitions.

Reply:
done